charts/nm-vllm-grafana-dashboards/templates/config-map-list.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: v1
+kind: ConfigMapList
+items:
+{{- $files := .Files.Glob "dashboards/*.json" }}
+{{- range $path, $fileContents := $files }}
+{{- $dashboardName := regexReplaceAll "(^.*/)(.*)\\.json$" $path "${2}" }}
+- apiVersion: v1
+  kind: ConfigMap
+  metadata:
+    name: {{ template "nm-vllm-grafana-dashboards.fullname" $ }}-grafana-dashboard
+    labels:
+      {{- if $.Values.label }}
+      {{ $.Values.label }}: {{ ternary $.Values.labelValue "1" (not (empty $.Values.labelValue)) | quote }}
+      {{- end }}
+  data:
+    {{ $dashboardName }}.json: {{ $.Files.Get $path | toJson }}
+{{- end }}
+
+# https://github.com/prometheus-community/helm-charts/blob/main/charts/kube-prometheus-stack/templates/grafana/configmap-dashboards.yaml
```

Comment:
I want this comment, but it feels awkward here. 😕

Reply:
I think I moved it to see if keeping it at the top was causing some parsing error (since someone on a github issue thread mentioned that somewhere), but I don't remember thinking the change was needed, would it being at the top with a small description feel right?